### ISSUE TITLE
Groundtruth pose of Real X-ray images

### ISSUE THREAD

**Issue by TUYaYa1** [gaocong13/Projective-Spatial-Transformers#29]
Hi, I recently encountered a problem when using real X-ray images. How can Groundtruth pose be obtained when using real X-ray images? Is it to randomly select a pose parameter and treat it as Groundtruth pose ?

**Comment by gaocong13** [comment]
No, the real X-ray ground truth poses are calibrated from the inserted metallic BBs in the specimen. The corresponding 2D and 3D BBs are annotated manually and the pose is solved from the pnp problem. This part of the code is not shared public available.

**Comment by TUYaYa1** [comment]
Ok, I see. Thank you
